app/src/main/res/values-de/strings.xml

```diff
@@ -345,13 +345,13 @@
     <string name="send_address_no_dnssec">OpenAlias ohne DNSSEC - Adresse kann gefälscht sein!</string>
     <string name="send_address_hint">Empfänger XMR/BTC Adresse oder OpenAlias</string>
 
-    <string name="progress_nfc_write">Writing Tag</string>
-    <string name="nfc_write_failed">Writing Tag failed!</string>
-    <string name="nfc_write_successful">Writing Tag successful</string>
-    <string name="nfc_tag_unsupported">Tag does not support NDEF!</string>
-    <string name="nfc_tag_size">Tag provides %d bytes, but we need %d!</string>
-    <string name="nfc_tag_read_undef">I don\'t understand the Tag!</string>
-    <string name="nfc_tag_read_what">I don\'t know what you want!</string>
-    <string name="nfc_tag_read_success">Reading Tag successful</string>
-    <string name="nfc_tag_tap">NFC Available!</string>
+    <string name="progress_nfc_write">NFC-Tag wird beschrieben</string>
+    <string name="nfc_write_failed">Beschreibung fehlgeschlagen!</string>
```

Comment:
349: "Schreiben fehlgeschlagen!" is also good. I assumed the ambiguity of "Beschreiben" (write / describe) is low in this scenario, since it is unlikely that the app writes some kind of process description for any reason. But nonetheless, "Schreiben" is shorter and less ambiguous.

Reply:
I'd understand "NFC-Tag wird beschrieben", but maybe you'd like "NFC-Tag wird geschrieben" better.

and "NFC-Tag konnte nicht geschrieben werden." for the other one.

I would personally prefer "beschrieben" though since it is common usage for discs and floppies.